.nvmrc

```diff
@@ -0,0 +1 @@
+v16.13.0
```

Comment:
no need for a precise version
```suggestion
16
```

Reply:
I just tested with your proposition, and it works 👍🏻